### ISSUE TITLE
Download speed based on categories

### ISSUE THREAD

**Issue by woiza** [nzbgetcom/nzbget#80]
Nzbget allows to define/limit download speed. I use this since basically everything is automated, runs "in the background", and I do not want to hinder other household members (saturate the internet connection).
However, every now and the I want to get a file as quickly as possible (no dl speed limitation or a higher dl speed limitation).
Would it be possible to implement dl speed based on categories?

**Comment by luckedea** [comment]
It can be a "priority" setting for Categories, good idea.
You say "a file as quickly as possible" - you already can do that with Priority for a specific download/file.

**Comment by woiza** [comment]
_you already can do that with Priority for a specific download/file._
How can I achieve this? Wouldn't this download still be throttled/limited by nzbget's global speed limit?

**Comment by luckedea** [comment]
No, it won't. Even Force Priority level won't go over speed limit. Priority only makes sure those specific nzbs would be downloaded in higher priority over other downloads.

Priority in NZBGet is supported for individual downloads, via external calls (can be set by Sonarr/Radarr for example), or per-feed, or managed by a Scan script. So there are options to manage download speeds.

For your specific case there - specific files would still be saturating the internet connection. That's why I assumed you'd prefer Priority management, rather than speed limit override. It's not hard to implement and it can be an extra option "Allow Force Priority downloads ignore Speed Limit".

**Comment by woiza** [comment]
_"Allow Force Priority downloads ignore Speed Limit"_ exactly that, thank you.

**Comment by luckedea** [comment]
v24 has been released, SpeedControl is prepared and would become available via Extension Manager next week. 
SpeedControl doesn't directly address this proposal, so keeping this issue open.
